napari/_qt/qt_main_window.py

```diff
@@ -129,6 +132,14 @@ def _add_window_menu(self):
         self.window_menu = self.main_menu.addMenu('&Window')
         self.window_menu.addAction(exit_action)
 
+    def _add_about_menu(self):
+        about_action = QAction('About', self._qt_window)
```

Comment:
Having 'About' as an action name without having `self.main_menu.setNativeMenuBar(False)` prevents adding help menu on mac osx. For now will be putting something else.